brainstormforce/ultimate-addons-for-gutenberg
# Fix: Instead of entry-content use body tag to hide the uag blocks on the page

[Trello](https://trello.com/c/sAlRm42X/262-fix-instead-of-entry-content-use-body-tag-to-hide-the-uag-blocks-on-the-page)

### Description
Right now we have added css to hide the block with the entry-content class -->  https://share.getcloudapp.com/ApuYOmRE
Every theme doesn't have this class so css will not work in this case. 
To resolve this we need to add body tag instead of entry-content class.

### Test cases
1. Add any block into the editor
2. Hide that block using display conditions.
3. check whether it is working properly or not.
4. check using various themes

### Types of changes
- Replaced entry-content class with body tag.

### How has this been tested?
Checked whether it is working properly or not on front-end.

### Checklist:
